### PR TITLE
Add Default user data (logging) + configure ecs agent

### PR DESCRIPTION
**Pull request by timpur**

### CODE REVIEW

File: packages/@cdk-cosmos/core/src/features/ecs-feature/ecs-feature-core-stack.ts [hunk 1]
```diff
@@ -151,13 +153,25 @@ export class EcsFeatureCoreStack extends BaseFeatureStack implements IEcsFeature
...
+    this.clusterAutoScalingGroup.userData.addCommands(
+      'cat <<EOF >> /etc/ecs/ecs.config',
+      ...Object.entries(config).map(([k, v]) => `${k}=${v}`),
+      'EOF'
+    );
+  }
 }
 
 const configureListener = (listener: ApplicationListener, listenerInboundCidr?: string | null): void => {
-  listener.addFixedResponse('Default', {
```
Comment (timpur) [inline packages/@cdk-cosmos/core/src/features/ecs-feature/ecs-feature-core-stack.ts:157]: deprecated

File: packages/@cdk-cosmos/core/src/features/ecs-feature/ecs-feature-core-stack.ts [hunk 2]
```diff
@@ -180,3 +194,15 @@ SolarSystemCoreStack.prototype.addEcs = function(props?: EcsSolarSystemCoreStack
...
+
+declare module '@aws-cdk/aws-ecs/lib/cluster' {
+  export interface AddCapacityOptions {
+    userData: UserData;
+  }
+}
+
+const defaultUserData = (): UserData => {
+  const userData = UserData.forLinux();
+  userData.addCommands('exec > >(tee /var/log/user-data.log|logger -t user-data -s 2>/dev/console) 2>&1');
```
Comment (timpur) [inline packages/@cdk-cosmos/core/src/features/ecs-feature/ecs-feature-core-stack.ts:206]: log out user data 
https://aws.amazon.com/premiumsupport/knowledge-center/ec2-linux-log-user-data/

**Comment by jbnaik** [comment]
Have you verified if that userdata line is actually redirecting output to the console..?

**Comment by timpur** [comment]
yeah ive used it before

**Comment by timpur** [comment]
also the add config functions are coming from tests in other accounts

**Pull request merged**